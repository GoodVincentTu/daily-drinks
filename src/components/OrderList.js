import React from 'react';
import OrderItem from './OrderItem';

const OrderList = (props) => {
  return (props.drinks &&
    <ul>
      <p>Orders</p>
      {props.drinks.map(drink => (
        <OrderItem key={drink.title} drink={drink}/>
      ))}
    </ul>
  )
};

export default OrderList;