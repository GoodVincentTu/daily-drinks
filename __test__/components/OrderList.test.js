import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import OrderList from '../../src/components/OrderList';

const fakeDrinks = [{
  title: '熟成紅茶',
  price: 20,
  notes: '微糖微冰'
}];

describe('<OrderList/>', () => {
  it('renders', () => {
    shallow(<OrderList drinks={fakeDrinks}/>);
  });

  it('matches the snapshot', () => {
    const wrapper = shallow(<OrderList/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
})