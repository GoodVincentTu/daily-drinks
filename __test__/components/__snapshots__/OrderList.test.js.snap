// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderList/> matches the snapshot 1`] = `""`;
